nosql/assignment.nosql.schema.txt

```diff
@@ -0,0 +1,84 @@
+
+Collection - Users
+
+{
+	account: {
+		username: "shirish.sp",
+		password: "**********",
+		roles: [
```

Comment:
- User will have only one role, user cannot be an admin and super admin at the same time